src/handle_provide_parameter.c

```diff
@@ -413,21 +441,29 @@ static void handle_governor_propose(ethPluginProvideParameter_t *msg, context_t
             context->next_param = VALUE;
             break;
         case VALUE:
-            copy_parameter(context->lisk.body.governorPropose.data.second[counter].value,
-                           msg->parameter,
-                           INT256_LENGTH);
-            if (counter + 1 < context->lisk.body.governorPropose.value_len) {
-                counter++;
-                context->next_param = SECOND_VALUE;
+            if (counter < context->lisk.body.governorPropose.value_len) {
```

Comment:
I think it's better to make it early return for readability
```
if (counter >= context->lisk.body.governorPropose.value_len) {
  msg->result = ETH_PLUGIN_RESULT_ERROR;
  break;
}
```

Reply:
Good point, thank you.

Done.